packages/aws-client/tests/test-DynamoDb.js

```diff
@@ -100,20 +101,25 @@ test.serial('DynamoDb.get() throws general error from failure on client.get', as
 
 test.serial('DynamoDb.scan() properly returns all paginated results', async (t) => {
   const { client } = t.context;
-  let count = 0;
-  const total = 3;
 
-  while (count < total) {
-    // eslint-disable-next-line no-await-in-loop
-    await client.put({
+  const items = range(3).map(() => ({
```

Comment:
Given we're already importing lodash, this is a tidy pattern in lieu of JS natively supporting 'do x N times' for small iteration counts.  👍🏻